The task findSonatypeStagingRepository is failing to match

```
> Task :findSonatypeStagingRepository FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':findSonatypeStagingRepository'.
> No staging repositories found for stagingProfileId: ***, descriptionRegex: \b\Q:spring-security-cas:4.0.0\E(\s|$). Here are all the repositories: [ReadStagingRepository(repositoryId=orggrails-1346, type=closed, transitioning=false, description=:spring-security-cas:4.0.0)]

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```

--- a/build.gradle
+++ b/build.gradle
@@ -16,6 +16,7 @@ repositories {
     maven { url "https://repo.grails.org/grails/core" }
 }
 
+group = "org.grails"
 version project.projectVersion
 
 ext {

